### DISCUSSION TITLE
Chat client plugin

### DISCUSSION THREAD

**Discussion by khs26** [zellij-org/zellij#141] [Ideas]
I realised that one obvious candidate for a plugin would be a chat client, thoughts?

**Comment by imsnif** [comment]
Would this be chat between people connected to the same session, or connect to some chat service, or?

**Comment by khs26** [comment]
Both I think. :) I was mainly thinking about external chat services (e.g. if we were doing co-dev, it'd be nice to have one pane with source, one pane with chat)

**Reply by SaintFenix** [reply]
for inter-session chat, do you mean something like [Talk](https://en.wikipedia.org/wiki/Talk_(software))?

**Comment by khs26** [comment]
Yes. I wasn't familiar with it, but it's not too surprising.
